installer/webapp/pom.xml

```diff
@@ -19,6 +19,108 @@
 
     <name>VIVO Install Web App</name>
 
+    <profiles>
```

Comment:
This was not a clean merge coming from `main` into `sprint-i18n`: 
- https://github.com/vivo-project/VIVO/blob/3e26e5657696b809efc3dd23aac9699fae7964aa/installer/webapp/pom.xml#L182-L192

I will have a follow-on to this PR in the next few days.

Reply:
Additional pom.xml / merge clean-up performed in commit below.